GTBitsOfGood/canine-assistants
# Create New Log Frontend

## Create New Log Frontend Modal
Issue Number(s): #32

The modal to create a new log has been created including the view for mobile web. The DropdownMenu component has been altered as well to allow for the option to choose single select, hide the Apply Filters button, hide checkboxes, and change the color for options selected.

### Checklist

### How To Test
ON WEB: go to `/dogs` and click on a dog then the "Logs" tab and then "+Add a log", create a successful log if every required field is filled out and the log shows an error state for each input when a required field is not filled out

ON MOBILE: Can either use Chrome's DevTools to view different mobile sizes, or run on a phone by first running on a computer, then type `ipconfig` and see the computer's IP address under Wi-Fi, then on a phone open the browser and go to `[IP_ADDRESS]:3000/dogs` and then follow the steps for web
